src/main/java/com/uniqueAuction/web/response/CommonResponse.java

```diff
@@ -0,0 +1,36 @@
+package com.uniqueAuction.web.response;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.uniqueAuction.exception.ErrorCode;
+import lombok.Getter;
+
+@JsonInclude(JsonInclude.Include.NON_DEFAULT)
+@Getter
+public class CommonResponse<T> {
+
+    private int code;
+    private String message;
+    private T data;
+
+
+    public CommonResponse(String message) {
+        this.message = message;
+    }
+
+    public CommonResponse(ErrorCode errorCode) {
+        this.code = errorCode.getCode();
+        this.message = errorCode.getMessage();
+    }
+
+    public static CommonResponse success(ErrorCode errorCode) {
+        return new CommonResponse(errorCode.getMessage());
+    }
+
+    public static CommonResponse fail(ErrorCode code) {
```

Comment:
```suggestion
    public static CommonResponse fail(ErrorCode errorCode) {
```

class 내부 필드인 code와 이름이 같기때문에 혼란을 줄수있는 이름이라 수정하는게 좋아보입니다
25번라인과의 통일성 측면에서도 변경이 더 좋아보여요

Reply:
넵 참고하겠습니다.